### ISSUE TITLE
Dock manager throws errors with Vite-based builds

### ISSUE THREAD

**Issue by ChronosSF** [IgniteUI/igniteui-dockmanager#88]
## Description  
Dock manager throws errors with Vite-based builds

## Steps to reproduce  
[DockManagerVite.zip](https://github.com/IgniteUI/igniteui-dockmanager/files/13317664/DockManagerVite.zip)

**Comment by igdmdimitrov** [comment]
As a temporary workaround while we further investigate you can modify the `vite.config.ts` to look like this:

```import { defineConfig } from 'vite'
import react from '@vitejs/plugin-react'

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()],
  optimizeDeps: {
    exclude: [
      'igniteui-dockmanager'
    ]
  }
})
```

**Comment by igdmdimitrov** [comment]
Fixed in v1.14.3
